http/templates/player_detail.html.tera

```diff
@@ -3,51 +3,78 @@
 {% import "macros" as macros %}
 {% block pagename %}{% if player %}{{ player.name }} Detail{% else %}Player not found{% endif %}{% endblock %}
 {% block body %}
-<div>
-    <span>{{ player.name }}</span>
-    {% if player.twitch_user_login %}
-    {% set url = "https://twitch.tv/<name>" | replace(from="<name>", to=player.twitch_user_login) %}
-    <div>{{ macros::external_link(href=url, text="Twitch stream") }}</div>
-    {% endif %}
-    {% if player.racetime_username %}
-    <div>Racetime username: {{ player.racetime_username }}</div>
-    {% endif %}
-    {% if player_history %}
-    <hr />
-    <div class="text-xl">
-        Race History
+<div id="player-detail-page" class="page-container">
+    <div class="page-title-container">
+        {% if pfp %}
+            {% set pfp_url = pfp %}
+        {% else %}
+            {% set pfp_url = "/static/images/placeholder-pfp.jpg" %}
+        {% endif %}
+        <div role="img" aria-label="User profile picture" id="pfp" style="background-image: url({{pfp_url}})"></div>
+
+        <div id="player-info-container">
+            <div id="player-name">{{ player.name }}</div>
+            <div id="player-info-badges-container">
+                {% if player.twitch_user_login %}
+                    <a href="https://twitch.tv/{{player.twitch_user_login}}" class="button player-icon twitch-icon">
+                        <img class="player-icon-image" src="/static/images/twitch-icon.png">
+                    </a>
+                {% endif %}
+                {% if player.racetime_username %}
+                    <a href="https://racetime.gg/user/{{player.racetime_user_id}}" class="button player-icon racetime-icon">
+                        <img class="player-icon-image" src="/static/images/racetime-icon.png">
+                    </a>
+                {% endif %}
+            </div>
+        </div>
     </div>
-    {% for season in player_history.seasons %}
 
-    <h2>{{ macros::link(href=season.season.url, text=season.season.title)}}</h2>
-    <div class="races">
-        <table>
-            <thead>
-                <tr>
-                    <td>Outcome</td>
-                    <td>Time</td>
-                    <td>
-                        Opponent
-                    </td>
-                    <td>
-                        Opponent's Time
-                    </td>
-                </tr>
-            </thead>
-            <tbody>
-                {% for race in season.races %}
-                <tr>
-                    <td>{{race.outcome}}</td>
-                    <td>{{race.time}}</td>
-                    <td>{{race.opponent}}</td>
-                    <td>{{race.opponent_time}}</td>
-                </tr>
-                {% endfor %}
+    <div class="page-content-container">
+        {% if player_history %}
```

Comment:
this is probably my fault for the specific structure of the data returned, but this is how it looks for a player who's in the DB but with no races. I think this is unimportant but I thought I'd mention it in case you felt strongly about it.

![image](https://github.com/FoxLisk/NMG-League-Bot/assets/1173225/272ee5f7-d1b8-4705-b2a8-5dec65258454)

Reply:
if you want to "fix" this (i.e. remove the `Race History` section entirely), you can change this to `{% if player_history.seasons %}`